README.md

```diff
@@ -120,7 +120,8 @@ module "postgres" {
 | vault_secret.vault_kv_password_name | Secret key name in Vault kv path | string | "password" | no |
 | memory | Memory allocation for Postgres in MB | number | 428 | no |
 | cpu | CPU allocation for Postgres in MHz | number | 350 | no |
-
+| resource_proxy | Resource allocations for proxy | obj(number, number) |	{ cpu = 500, memory = 1024} |	no |
```

Comment:
I suggest the following if this comment gets mereged (https://github.com/fredrikhgrelland/terraform-nomad-postgres/pull/57/files#r522788761)
```suggestion
| resource_proxy | Resource allocations for proxy | obj(number, number) |	{ <br> cpu = 200, <br> memory = 128 <br> } |	no |
```

Reply:
We encountered an OOM error with this low amount of memory in foundation 2.

Reply:
Yes, but by adding the resources as variables you can simply change it wherever you use the module

Reply:
I agree, but having a default value which is too low for the module to work doesn't make sense to me.

Reply:
The module works with that amount. Just not in foundation 2 apparently